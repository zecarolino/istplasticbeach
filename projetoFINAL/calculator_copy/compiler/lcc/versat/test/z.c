int x;
int main() {
	x = 0x38;
	asm("\tldi x\n\taddi MEM_BASE\n\trdwb\n\trdwb\n\twrw CPRT_BASE\n");
	asm("\tldi 0xa\n\twrw CPRT_BASE\n");
	return 0;
}
