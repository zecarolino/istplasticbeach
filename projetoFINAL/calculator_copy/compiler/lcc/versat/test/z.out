VCD info: dumpfile xtop.vcd opened for output.
8
0
Catched Trap at time  398 clock cycles:
